docs/source/conf.py

```diff
@@ -322,3 +324,21 @@ def get_incollection_template(self, e):
 
 # display logo on top-left of html pages
 html_logo = "img/unitary_fund_logo.png"
+
+myst_update_mathjax = False
+
+nbsphinx_custom_formats = {
+    '.mystnb': ['jupytext.reads', {'fmt': 'mystnb'}],
+}
+nbsphinx_execute = 'always'
+
+nbsphinx_thumbnails = {
+    'examples/ibmq-backends': '_images/unitary_fund_logo.png',
+    'examples/hamiltonians': '_images/vqe-cirq-pauli-sum-mitigation-plot.png',
+    'examples/simple_landscape': '_images/simple_landscape_13_0.png',
+    'examples/maxcut-demo': '_images/maxcut-demo_48_1.png',
+    'examples/pyquil_demo': '_images/unitary_fund_logo.png',
+    'examples/cdr_api': '_images/unitary_fund_logo.png',
+    'examples/mitiq-paper/mitiq-paper-codeblocks':
+        '_images/unitary_fund_logo.png',
+}
```

Comment:
I think you need to add these images?

Q: What happens if we add a new example and don't add an entry to this dict? Does it default to some standard image? Is it possible to default to `unitary_fund_logo.png`?

Reply:
This is better, thanks!